contracts/OctobayGovNFT.sol

```diff
@@ -19,14 +19,13 @@ contract OctobayGovNFT is OctobayStorage, ERC721Pausable {
 
     event MintNFTForGovTokenEvent(address to, uint256 tokenId, address govTokenAddress);
     event BurnTokenEvent(uint256 tokenId);
+    event TokenTransferEvent(address from, address to, uint256 tokenId);
```

Comment:
I think this isn't needed. https://github.com/OpenZeppelin/openzeppelin-contracts/blob/5cd86f740d9a4b351cad196e7957a7d0406e7368/contracts/token/ERC721/IERC721.sol#L14